frontmatter.js

```diff
@@ -0,0 +1,74 @@
+const matter = require("gray-matter");
```

Comment:
what is this file for?

Reply:
documentation rendering so I can do `mdx` files